_common.R

```diff
@@ -92,7 +92,7 @@ Bstratum.labels <- c(
   "Age < 65, Not at risk"
 )
 
-llods <- c(bindSpike=20, bindRBD=20, pseudoneutid50=10, pseudoneutid80=10) 
+llods <- c(bindSpike=20, bindRBD=20, pseudoneutid50=10, pseudoneutid80=10)
```

Comment:
please use the LLODs already included in `_common.R`, https://github.com/CoVPN/correlates_reporting/blob/master/_common.R#L95